network-api/networkapi/wagtailpages/templatetags/bg_selector_tags.py

```diff
@@ -14,17 +14,6 @@ def yes_no(value):
     return gettext('Unknown')
 
 
-@register.filter
-def extended_boolean(value):
-    if value == 'Yes':
-        return gettext('Yes')
-    if value == 'No':
-        return gettext('No')
-    if value == 'U':
-        return gettext('Unknown')
-    return value
-
-
 @register.filter
 def extended_yes_no(value):
```

Comment:
Since the only different option in extended_boolean is "U" we can use add it to "extended_yes_no" instead, and this will allow us to parse all values using the extended_yes_no tag in "product_criterion_primary"

Reply:
good thinking!